### PR TITLE
Add logic to convert km name to uuid

### PR DESCRIPTION
**Pull request by Arshardh**
## Automation tests
 - Unit tests 
   > Code coverage information
 - Integration tests
   > Details about the test cases and coverage

## Security checks
 - Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? yes/no
 - Ran FindSecurityBugs plugin and verified report? yes/no
 - Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? yes/no

### CODE REVIEW

File: migration-client/wso2-api-migration-client/src/main/java/org/wso2/carbon/apimgt/migration/dao/APIMgtDAO.java [hunk 1]
```diff
@@ -586,6 +623,108 @@ public boolean isCrossTenantAPISubscriptionsExist(TenantManager tenantManager) t
...
+        try (Connection connection = APIMgtDBUtil.getConnection()) {
+            connection.setAutoCommit(false);
+            HashMap<String, String> results = new HashMap<>();
+            try (PreparedStatement preparedStatement = connection.prepareStatement(GET_KEY_MAPPING)) {
+                preparedStatement.setInt(1, tenant.getId());
+                preparedStatement.setString(2, tenant.getDomain());
+                try (ResultSet resultSet = preparedStatement.executeQuery()) {
+                    connection.commit();
+                    while (resultSet.next()) {
+                        keyID = resultSet.getString("KEY_ID");
```
Comment (tharindu1st) [inline migration-client/wso2-api-migration-client/src/main/java/org/wso2/carbon/apimgt/migration/dao/APIMgtDAO.java:644]: move 
String keyID = null;
  String keyManagerUUID = null;
into here

File: migration-client/wso2-api-migration-client/src/main/java/org/wso2/carbon/apimgt/migration/dao/APIMgtDAO.java [hunk 2]
```diff
@@ -586,6 +623,108 @@ public boolean isCrossTenantAPISubscriptionsExist(TenantManager tenantManager) t
...
+                        keyManagerUUID = resultSet.getString("KEY_MANAGER_ID");
+                        results.put(keyID, keyManagerUUID);
+                    }
+                }
+            }
+            updateKeyMappings(connection, results);
```
Comment (tharindu1st) [inline migration-client/wso2-api-migration-client/src/main/java/org/wso2/carbon/apimgt/migration/dao/APIMgtDAO.java:650]: missing connection.rollBack

**Review by tharindu1st — CHANGES_REQUESTED** [review]

File: migration-client/wso2-api-migration-client/src/main/java/org/wso2/carbon/apimgt/migration/dao/APIMgtDAO.java (hunk 1 above)
Comment (Arshardh) [reply migration-client/wso2-api-migration-client/src/main/java/org/wso2/carbon/apimgt/migration/dao/APIMgtDAO.java]: Fixed with 3c59eccb8a5f25e069978a6132e87c30a8d016ec

File: migration-client/wso2-api-migration-client/src/main/java/org/wso2/carbon/apimgt/migration/dao/APIMgtDAO.java (hunk 2 above)
Comment (Arshardh) [reply migration-client/wso2-api-migration-client/src/main/java/org/wso2/carbon/apimgt/migration/dao/APIMgtDAO.java:651]: Fixed with 3c59eccb8a5f25e069978a6132e87c30a8d016ec

**Review by tharindu1st — APPROVED** [review]

**Pull request merged**